configure.ac

```diff
@@ -153,16 +152,15 @@ AC_SUBST(PKGCONFIG_VERSION)
 ##########################################################################
 AC_ARG_WITH([platform],
     [AS_HELP_STRING([--with-platform=platform],
-	[select platform to be used, default linux-dpdk])],
+	[select platform to be used, default linux-generic])],
     [],
-    [with_platform=linux-dpdk
+    [with_platform=linux-generic
     ])
 
 AC_SUBST([with_platform])
 AC_SUBST([platform_with_platform], ["platform/${with_platform}"])
 
 ##########################################################################
-<<<<<<< HEAD
```

Comment:
@lumag see commit 2: https://github.com/Linaro/odp/pull/123/commits/78eacc5eec31e41f82eba318d0030e8c3778bfed First commit just does raw marge with all rejects present. Second commit removes rejected things. So it will be more easy to review that merge. On apply 2 patches can be squashed if needed.